src/Core/Components/Overflow/FluentOverflow.razor.js

```diff
@@ -1,7 +1,9 @@
 let resizeObserver;
 let observerAddRemove;
+let lastId;
```

Comment:
Why did you need these variables?

`lastId` will be overwritten when several **FluentOverflow** are used at the same time, as it is a single variable for all components. Is this correct?

Reply:
I have tested and confirmed this is working as it should (added OverflowDefault example to AppBarDefault example). Is is only of interest if and when the orientation of an AppBar is changed which will typically not happen as frequently. In the example we now provide that option explicitly but that is not the case for regular usage, I think.